Pixi/Fable.Import.Pixi.fsproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>0.1.0</Version>
```

Comment:
You need to set this version to 0.0.0.

The version will be updated by the `build.fsx` according to the RELEASE_NOTES.md version.